### DISCUSSION TITLE
Switch wall for power consumption

### DISCUSSION THREAD

**Discussion by DimitriXav** [petrleocompel/hikaxpro_hacs#81] [General]
Hi,

Is it possible to add the wall switches (DS-PM1-O1H-WB)? be able to use the data to have electricity consumption in home assistant.

Thanks

![Screenshot_20240108-075922](https://github.com/petrleocompel/hikaxpro_hacs/assets/95666539/0948f98f-6ce0-438f-99f7-1977cf56a914)

**Comment by petrleocompel** [comment]
Yes, most important would be provide logs and testing time. Because I do not own every single device from HikVision. 
I am unsure about the "statistic" due pull times. But if API has some endpoint it can be managed. 
It is kinda request of #31  and for separate devices we are creating separate issues.
